echo.go

```diff
@@ -633,8 +633,8 @@ func (e *Echo) Routes() []*Route {
 
 // AcquireContext returns an empty `Context` instance from the pool.
 // You must return the context by calling `ReleaseContext()`.
-func (e *Echo) AcquireContext() Context {
```

Comment:
What is the difference between returning `interface` and `pointer -> struct`?

Reply:
You can assign `*echo.context` to `echo.Context` because that struct satisfies that interface. but `AcquireContext` is a **public** method than anyone can use  and now changing it would/could make their code do not compile anymore because you can not assign other structs (like you own custom context implementation) to that  struct (`*echo.context`)

```go
	c := e.AcquireContext()
	e.ReleaseContext(c)
	c = new(MyCustomContext) // Cannot use 'new(MyCustomContext)' (type *MyCustomContext) as the type *context
```

Yes, this is hypotetical but it still breaks library user code, it does not compile even anymore

Reply:
I see, I understand it well. Thank you very much.